src/en/mangadistrict/src/eu/kanade/tachiyomi/extension/en/mangadistrict/MangaDistrict.kt

```diff
@@ -29,80 +29,71 @@ class MangaDistrict :
         return super.popularMangaFromElement(element).apply {
-            if (preferences.getBoolean(REMOVE_TITLE_COMMENT_PREF, true)) {
-                title = this.title.replace(titleComment, "").trim()
+            if (isRemoveTitleVersion()) {
+                title = this.title.replace(titleVersion, "").trim()
             }
         }
     }
 
     override fun searchMangaFromElement(element: Element): SManga {
         return super.searchMangaFromElement(element).apply {
-            if (preferences.getBoolean(REMOVE_TITLE_COMMENT_PREF, true)) {
-                title = this.title.replace(titleComment, "").trim()
+            if (isRemoveTitleVersion()) {
+                title = this.title.replace(titleVersion, "").trim()
             }
         }
     }
 
     override fun mangaDetailsParse(document: Document): SManga {
         return super.mangaDetailsParse(document).apply {
-            if (preferences.getBoolean(REMOVE_TITLE_COMMENT_PREF, true)) {
-                title = this.title.replace(titleComment, "").trim()
+            if (isRemoveTitleVersion()) {
+                title = this.title.replace(titleVersion, "").trim()
             }
         }
     }
 
     override fun chapterListParse(response: Response): List<SChapter> {
-        return super.chapterListParse(response).filterNot {
-            when (getImgResPref()) {
-                "Full quality" -> it.url.contains("/v1-high-quality")
-                "High quality" -> it.url.contains("/v2-full-quality")
-                else -> false
-            }
+        val chapters = super.chapterListParse(response)
+        return when (getImgRes()) {
+            IMG_RES_HIGH -> chapters.filterNot { it.url.contains("/v2-full-quality") }
+            IMG_RES_FULL -> chapters.filterNot { it.url.contains("/v1-high-quality") }
+            else -> chapters
         }
     }
 
-    private fun getImgResPref(): String? = preferences.getString(IMG_RES_PREF_KEY, IMG_RES_PREF_DEFAULT_VALUE)
+    private fun isRemoveTitleVersion() = preferences.getBoolean(REMOVE_TITLE_VERSION_PREF, false)
+    private fun getImgRes() = preferences.getString(IMG_RES_PREF, IMG_RES_DEFAULT)!!
 
     override fun setupPreferenceScreen(screen: PreferenceScreen) {
         SwitchPreferenceCompat(screen.context).apply {
-            key = REMOVE_TITLE_COMMENT_PREF
-            title = "Remove comments from title, e.g. \"(Official)\", \"(Doujinshi)\""
-            summary = "To help app find duplicate entries in library"
-            setDefaultValue(true)
+            key = REMOVE_TITLE_VERSION_PREF
+            title = "Remove version information from entry titles"
+            summary = "This removes version tags like “(Official)” or “(Doujinshi)” from entry titles " +
+                "and helps identify duplicate entries in your library. " +
+                "To update existing entries, remove them from your library and clear database in advanced settings."
```

Comment:
No need to remove entry from library, just need to refresh each entry to update its title.

Reply:
The app won't update the titles of entries in the library (i.e. favorited). Clearing database is not necessary indeed.